main.tex

```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
 develop, and contribute to those tools within a conscientious and welcoming
 community.
 
-\acknowledgments
-
-Who to thank?  numfocus, GSOC, Python Software Foundation
-
-We thank the many web services that make developing and maintaining this package
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students ofter turn into long-term contributors. We also
+thank NumFOCUS and the Python Software Foundation for financial
```

Comment:
Fairly trivial, but my feeling is this should be mentioned before summer of code

Reply:
I sorted in rough order of funding. I don't know exactly got much we got from NumFOCUS and the PSF, but we've had GSOC students for a few years not with 5-10 students per year. Let's say 5 years (not sure) * (5 students * 5000 $ + mentor summit + org payments) > 150 k$.